### PR TITLE
Fix 2j3 wording

### PR DESCRIPTION
**Pull request by Ivan-Brigidano**

### CODE REVIEW

**Comment by xsrvmy** [comment]
The first "sentence" of 2j3 is still a fragmented sentence so that needs to be fixed.
The current phrasing allows it to be the continuation of 2j with a comma, but that does not line up with 2j1 and 2j3.

**Comment by Ivan-Brigidano** [comment]
> The first "sentence" of 2j3 is still a fragmented sentence so that needs to be fixed.
> The current phrasing allows it to be the continuation of 2j with a comma, but that does not line up with 2j1 and 2j3.

@xsrvmy I see what you mean. I'll try to rephrase it to make it clear

File: wca-regulations.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 2j1) If a competitor is disqualified from an event for any reason, they are not eligible for any more attempts in the event.
         - 2j1a) The results of all remaining attempts in the event are recorded as DNF.
     - 2j2) If a competitor is disqualified during the course of an event, their earlier results remain valid. Exception: cheating or defrauding (see [Regulation 2k2a](regulations:regulation:2k2a)).
-    - 2j3) If a competitor does not fulfill the event's requirements (e.g. know how to solve the puzzle). A competitor must not compete with the expectation of a DNF result or an intentionally poor result.
+    - 2j3) If a competitor does not fulfill the event's requirements (e.g. not knowing how to solve the puzzle). A competitor must not compete with the expectation of a DNF result or an intentionally poor result.
```
Comment (Ivan-Brigidano) [inline wca-regulations.md:93]:
```suggestion
    - 2j3) A competitor may be disqualified if the competitor does not fulfill the event's requirements (e.g. not knowing how to solve the puzzle). A competitor must not compete with the expectation of a DNF result or an intentionally poor result.
```

Comment (Ivan-Brigidano) [reply wca-regulations.md:93]: @KatieHull @Sydney-Weaver (or any other native English speaker) can you review this?

**Comment by Laura-O** [comment]
This is an extremely weird change and I do not understand why you want to move it here:
2j is about how disqualifications are handled, i.e. if results are kept, etc.
The place where you probably want to move this is 2k, which is about reasons for disqualifications.

**Comment by Ivan-Brigidano** [comment]
> This is an extremely weird change and I do not understand why you want to move it here:
> 2j is about how disqualifications are handled, i.e. if results are kept, etc.
> The place where you probably want to move this is 2k, which is about reasons for disqualifications.

@Laura-O yes, that's right. I'll make the appropriate changes.
